NetExercise/Basic/Exercise18.cs

```diff
@@ -0,0 +1,40 @@
+namespace NetExercise.Basic
+{
+    public class Exercise18
+    {
+        public static bool HappyNumber(int n)
+        {
+            while ((n != 1) && (n != 4))
+            {
+                n = Convert(n);
+            }
+
+            return n == 1;
+        }
+        
+        private static int Convert(int n)
+        {
+            var temp = 0;
+            
+            while (n > 0)
+            {
+                temp += Power(n % 10, 2);
```

Comment:
Dùng dao mổ trâu để thịt gà. Ở đây luôn tình bình phương, viết thế này cho nhanh khỏi
```suggestion
                temp += (n % 10) * (n % 10);
```

Reply:
hehe,quên mất hè :),thanks e